flaml/automl/automl.py

```diff
@@ -3557,6 +3569,13 @@ def _search_sequential(self):
                     state.best_config,
                     self.data_size_full,
                 )
+
+                if (
+                    self._trained_estimator.params[self._trained_estimator.ITER_HP]
+                    != self.best_config[self._trained_estimator.ITER_HP]
+                ):
+                    logger.warning("early stopping happened")
```

Comment:
Check the code coverage. If not covered, add a test case. Also, change the warning message as it is ambiguous.